doc-site/docs/releasenotes/index.md

```diff
@@ -2,8 +2,29 @@
 title: Release Notes
 ---
 
+# Release Notes
+
 [Full release notes](https://github.com/hyperledger/firefly/releases)
 
+## [v1.3.3 - Mar 24, 2025](https://github.com/hyperledger/firefly/releases/tag/v1.3.3)
+
+What's New:
+- Add new method to provide reliable receipt storage for blockchain plugins
+- Changes in metrics:   
+    - Added new metrics for Data Exchange 
+    - Added a namespace label to existing metrics and change gauges to vectors
```

Comment:
This is two different bullets

Reply:
The latter is a result of the first, we need to change to a vector to add labels

Reply:
but I don't think we need to mention it here but instead in the migration consideration